### PR TITLE
Issue 1 Tasks Completed

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-02-08 at 7 22 42 PM
Screen Shot 2021-02-08 at 7 22 56 PM
Screen Shot 2021-02-08 at 7 31 12 PM

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+  event.preventDefault();
+  var newObject = {
+    imageUrl: $displayedImage.getAttribute('src'),
+    title: $title.value,
+    notes: $notes.value,
+    nextEntryId: data.nextEntryId
+  };
+  data.entries.unshift(newObject);
+  var entriesJSON = JSON.stringify(data.entries);
+  localStorage.setItem('code-journal-local-storage', entriesJSON);
```
Comment (codingcodymiller) [inline js/main.js]: This should be in the `data.js` file. Also, the value being stored in localStorage should be the entire `data` object, not just the `entries` property. Lastly, this should be done before the page unloads, not after every submit event

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+var $notes = document.querySelector('#notes');
+var previousEntriesJSON = localStorage.getItem('code-journal-local-storage');
+
+$photoURL.addEventListener('input', function (event) {
+  $displayedImage.setAttribute('src', event.target.value);
+});
+
+if (previousEntriesJSON !== null) {
+  data.entries = JSON.parse(previousEntriesJSON);
+}
```
Comment (codingcodymiller) [inline js/main.js]: This should be in the `data.js` file. Also, the value being parsed from localStorage should be the entire `data` object, not just the `entries` property.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! Please see my notes.

Also, your commit messages thus far do not give any information on what you worked on. You should use the description of the step you are working on in the Issue for the Feature as your commit message. Commit messages should always describe what work was done in that commit.

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-09 at 12 10 52 PM

File: js/main.js [hunk 3]
```diff
@@ -5,27 +5,20 @@ var $displayedImage = document.querySelector('.displayed-image');
...
-}
-
 $form.addEventListener('submit', function (event) {
   event.preventDefault();
   var newObject = {
     imageUrl: $displayedImage.getAttribute('src'),
     title: $title.value,
     notes: $notes.value,
-    nextEntryId: data.nextEntryId
+    EntryId: data.nextEntryId
```
Comment (codingcodymiller) [inline js/main.js]: 🤮 

camelCase, please

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-09 at 12 19 38 PM

**Review by codingcodymiller — APPROVED** [review]
🎉 🎉 🎉 🎉

**Pull request merged**